src/components/InputField.js

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import { COLORS } from '../constants/COLORS';
+
+const InputField = ({ setItemList }) => {
+  const [inputText, setInputText] = useState('');
+
+  const handleInputChange = (e) => {
+    setInputText(e.target.value);
+  };
+
+  const handleFormSubmit = (e) => {
+    e.preventDefault();
+
+    if (inputText) {
```

Comment:
`inputText`가 공백문자여도 todo가 추가되네요. 이렇게 해보면 어떨까요?
```suggestion
    if (inputText.replace(/\s+/g, '')) {
```

Reply:
제가 방금 테스트 해 보니 로컬과 배포한 환경에서 모두 공백 문자 추가가 불가능한데 공백 문자가 추가되는 환경이 있나요?!?!?!?

Reply:
<img width="390" alt="image" src="https://user-images.githubusercontent.com/30287999/160247539-a70300fb-518d-45ef-875b-18d02f22042e.png">

스페이스바 한 두세개 넣고 엔터 누르면 투두가 생겨버리네요....

Reply:
와 그 경우는 제가 미처 생각하지 못했네요....... `''`일 때에만 테스트해 봐서 그랬군요........ 감사합니다... 👍🏻